overlord/hookstate/ctlcmd/get.go

```diff
@@ -73,9 +73,10 @@ Values of interface connection settings may be printed with:
     $ snapctl get :myplug usb-vendor
     $ snapctl get :myslot path
 
-This will return the named setting from the local interface endpoint, whether a plug
-or a slot. Returning the setting from the connected snap's endpoint is also possible
-by explicitly requesting that via the --plug and --slot command line options:
+This will return the named setting from the local interface endpoint, whether a
```

Comment:
```suggestion
This will return the named setting from the local interface endpoint, regardless of whether it's a
```
The original wording could imply it returns an additional value to indicate whether the interface endpoint is a plug or a slot.